### PR TITLE
[Breaking] Remove GA tracking

### PR DESCRIPTION
**Pull request by YahiaElTai**
#### Summary

This PR removes GA tracking from `appkit`.

Note: I removed `perfumejs` performance tracking as well as this was submitted to Google Analytics, I'm really not sure if what that was correct to remove it, please let me know and I can bring perfumejs back if needed 😅

### CODE REVIEW

File: packages/application-shell/src/components/navbar/menu-items.tsx [hunk 1]
```diff
@@ -146,11 +142,6 @@ const MenuExpander = (props: MenuExpanderProps) => {
     >
       <div
         onClick={(event) => {
-          const { action, category, label } =
-            trackingEvents[
-              props.isMenuOpen ? 'collapseMainNav' : 'expandMainNav'
-            ];
-          gtmTracking.track(action, category, label);
           props.onClick(event);
```
Comment (kark) [inline packages/application-shell/src/components/navbar/menu-items.tsx:145]: nit:
```suggestion
          onClick={props.onClick}
```

**Review by kark — COMMENTED** [review]
Thank you 🙏 
It looks good to me in general 🙌
I suppose we'd need to add changeset(s) as part of this PR.

**Comment by YahiaElTai** [comment]
Thanks for the review @kark , I have added the requested changes and waiting for your review 🥇

**Review by kark — APPROVED** [review]
Thank you again, all looks good to me 💪

@commercetools/shield-team-fe should we target `v22-release` branch (or something along those lines) like it was done [here](https://github.com/commercetools/merchant-center-application-kit/pull/2430/files) or use this one as a release branch?

**Review by emmenko — APPROVED** [review]
Thanks @YahiaElTai , I added some more info to the changeset but otherwise looks good 🥳

**Review by CarlosCortizasCT — APPROVED** [review]
💯

**Pull request merged**